src/main/java/frc/robot/subsystems/mailbox/MailboxBelts.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Asimov's Laws:
+ * The First Law: A robot may not injure a human being or, through inaction, allow a human being to come to harm.
+ * The Second Law: A robot must obey the orders given it by human beings except where such orders would conflict with the First Law.
+ * The Third Law: A robot must protect its own existence as long as such protection does not conflict with the First or Second Law.
+ */
+
+package frc.robot.subsystems.mailbox;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+/** Subsystem for the mailbox belt. */
+public class MailboxBelts extends SubsystemBase {
+  private CANSparkMax beltMotor;
+
+  /** Create an object called "beltMotor" */
```

Comment:
Constructor for the mailbox belts subsystem.

Generally, we want to describe what a given method does in the context that someone calling the method would need to know. Someone else calling the method doesn't care that it instantiates an object that represents the motor—they care that it's the constructor for the class.